legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -45,18 +45,26 @@ class EligibilityDetails:
         ar_overdue: bool
         transition_overdue: bool
 
+    @dataclass
+    class ExclusionSettings:
+        """Details about the exclude of a business for dissolution."""
+
+        exclude_in_dissolution: bool = True
+        exclude_future_effective_filing: bool = False
+
     @classmethod
     def check_business_eligibility(
-        cls, identifier: str, exclude_in_dissolution=True
+        cls, identifier: str, exclude_settings: ExclusionSettings = ExclusionSettings()
```

Comment:
This change also breaks [furnishings job](https://github.com/bcgov/lear/blob/main/jobs/furnishings/src/furnishings/stage_processors/stage_one.py#L55). Can you also modify the job if it's convenient for you?

Reply:
Yes, It will break. If this change is good, then, I will update other services too

Reply:
The updates for other BE services can be done in a separate PR so we can verify if tests pass.